### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marsamak**
Hi, I'm trying to accomplish my homework for the Lesson2 - here is my first PR ever. Please review it=)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# redi_frontend_a2020
\ No newline at end of file
+# redi_frontend_a2020
+> Separate repository for general practice in the class and regular homeworks of ReDi Frontend course (Autumn 2020)
```
Comment (MrTim) [inline README.md:2]: `>` thing means quote. Is it really what you aimed to? :)

**Review by MrTim — APPROVED** [review]
One small question about markup but rest is fine for me!

File: README.md (hunk 1 above)
Comment (marsamak) [reply README.md:2]: https://gist.github.com/fvcproductions/1bfc2d4aecb01a834b46
I took it here=) I thougth it is this nice looking vertical line=)) It's shown to me in that way, when I use the github online viewer=)

Comment (MrTim) [reply README.md:2]: Here is a nice reference: https://guides.github.com/features/mastering-markdown/#intro
Yeah if you want that => sure go for it! But in general if you will work with shared md files you will need to stick with regular markdown usage :)

**Pull request merged**